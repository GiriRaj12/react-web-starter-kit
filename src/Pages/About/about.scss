.basic-view{
    text-align: center;
    padding: 40px;
    font-size: large;
    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
    width: 50%;
    margin: 50px auto;
    box-shadow: 0 0 5px 0px rgb(238, 238, 238)
}