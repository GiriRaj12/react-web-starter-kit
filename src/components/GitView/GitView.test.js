import React from 'react';
import './about.scss';

function About() {
    return (<div className='basic-view'>
        Basic View
    </div>)
}

export default About;