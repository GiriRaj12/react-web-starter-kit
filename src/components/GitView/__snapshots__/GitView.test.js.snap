// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitView Snapshot test 1`] = `
<div
  className="basic-view"
>
  <p>
    Git link provided
  </p>
  <a
    href="https://github.com/GiriRaj12/react-web-starter-kit"
    rel="noopener noreferrer"
    target="_blank"
  >
    Click Here
  </a>
</div>
`;
